docs/v1.md

```diff
@@ -59,6 +60,23 @@
 - has(id: `string`) : `boolean`
 - isEmpty() : `boolean`
 
+## Components
+- servers(): `Servers`
+- channels(): `Channels`
+- messages(): `Messages`
+- schemas(): `Schemas`
+- channelParameters(): `ChannelParameters`
+- serverVariables(): `ServerVariables`
+- operationTraits(): `OperationTraits`
+- messageTraits(): `MessageTraits`
+- correlationIds(): `CorrelationId[]`
+- securitySchemes(): `SecuritySchemes`
+- serverBindings(): `Bindings`
+- channelBindings(): `Bindings`
+- operationBindings(): `Bindings`
+- messageBindings(): `Bindings`
```

Comment:
These methods has invalid return type. `Bindings` is a collection of bindings, but for single object: `channel`, `server`, `operation`, `message` - `Bindings` takes object like https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#server-bindings-object and transform it to the list of bindings with `protocol` as `id`. For these fields spec has signature as 

```
Map[string, [Server Bindings Object | Reference Object]`
```

so we need to change it to the:

```suggestion
- serverBindings(): `Record<string, Bindings>`
- channelBindings(): `Record<string, Bindings>`
- operationBindings(): `Record<string, Bindings>`
- messageBindings(): `Record<string, Bindings>`
```

Reply:
Good catch. I'm going to use `Map[string, Bindings]` per convention with what we decided originally on this API to use for maps.